src/tiledb/cloud/file/udfs.py

```diff
@@ -0,0 +1,88 @@
+from functools import partial
+from math import ceil
+from typing import Any, List, Mapping, Optional, Sequence, TypeVar
+
+from tiledb.cloud.file import utils as file_utils
+from tiledb.cloud.utilities import chunk
+from tiledb.cloud.utilities import find
+from tiledb.cloud.utilities import get_logger_wrapper
+
+DEFAULT_BATCH_SIZE = 100
+T = TypeVar("T")
```

Comment:
nit: to avoid these looking like they should be exportable, I tend to call them with an underscore, like `_T`.

Reply:
I think this is usually named that way in order to make sense to C/C++ engineers, but I get your point.